detox/e2e/test/messaging/message_local_drafts.e2e.ts

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *******************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element testID when selecting an element. Create one if none.
+// *******************************************************************
+
+import {
+    Post,
+    Setup,
+} from '@support/server_api';
+import {
+    serverOneUrl,
+    siteOneUrl,
+} from '@support/test_config';
+import {
+    ChannelScreen,
+    ChannelListScreen,
+    DraftScreen,
+    HomeScreen,
+    LoginScreen,
+    ServerScreen,
+} from '@support/ui/screen';
+import {getRandomId, isIos} from '@support/utils';
+import {expect} from 'detox';
+
+describe('Messaging - Message Draft', () => {
+    const serverOneDisplayName = 'Server 1';
+    const channelsCategory = 'channels';
+    let testChannel: any;
+
+    beforeAll(async () => {
+        const {channel, user} = await Setup.apiInit(siteOneUrl);
+        testChannel = channel;
+
+        // # Log in to server
+        await ServerScreen.connectToServer(serverOneUrl, serverOneDisplayName);
+        await LoginScreen.login(user);
+    });
+
+    beforeEach(async () => {
+        // * Verify on channel list screen
+        await ChannelListScreen.toBeVisible();
+    });
+
+    afterAll(async () => {
+        // # Log out
+        await HomeScreen.logout();
+    });
+
+    it('MM-T5637 should be able to send the draft message from Draft screen', async () => {
+        // # Open a channel screen and create a message draft
+        const message = `Message ${getRandomId()}`;
+        await openChannel(channelsCategory, testChannel);
+        await createDraft(message, testChannel);
+        await verifyDraftonChannelList('1');
+        await ChannelListScreen.draftsButton.tap();
+        await DraftScreen.draftTooltipCloseButton.tap();
+        await DraftScreen.openDraftPostActions();
+        await DraftScreen.sendDraft();
+        await DraftScreen.backButton.tap();
+    });
+
+    it('MM-T5638 should be able to swipe left and delete a draft message', async () => {
+        // # Open a channel screen and create a message draft
+        const message = `Message ${getRandomId()}`;
+        await openChannel(channelsCategory, testChannel);
+        await createDraft(message, testChannel);
+        await verifyDraftonChannelList('1');
+        await ChannelListScreen.draftsButton.tap();
+        await DraftScreen.swipeDraftPostLeft();
+        await DraftScreen.deleteDraftPostFromSwipeActions();
+        await DraftScreen.backButton.tap();
+    });
+
+    it('MM-T5638 should be able to delete a draft message from long press Draft actions', async () => {
```

Comment:
If test case in Zephyr has multiple steps, then let's indicate step number. Same with the other test case.